src/test/xmeta/test_symbol_table.cpp

```diff
@@ -517,3 +515,377 @@ TEST_CASE("Resolving types across namespaces test")
         REQUIRE(std::get<std::shared_ptr<enum_model>>(target)->get_id() == "E1");
     }
 }
+
+TEST_CASE("Interface base test")
+{
+    std::istringstream test_idl{ R"(
+        namespace N
+        {
+            interface IControl
+            {
+                void Paint();
+            }
+            interface ITextBox requires IControl
+            {
+                void SetText(String text);
+            }
+            interface IListBox requires IControl
+            {
+               void SetItem(String items);
+            }
+            interface IComboBox requires ITextBox, IListBox {}
```

Comment:
This case seems odd for two reasons:
1. I didn't think xlang was supporting multiple interface inheritance. And multiple "requires" isn't supported without some sort of attribute specifying which required interface becomes the base for inheritance purposes.
2. This makes IComboBox.Paint() ambiguous. This should require some sort of disambiguation on the method.

Reply:
I see what you're saying. I can disable this test for now and come back to it later once we have attributes.

Reply:
And also add a comment